pkgs/by-name/ku/kubectl-get-all/package.nix

```diff
@@ -0,0 +1,32 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+
+buildGoModule rec {
+  pname = "kubectl-get-all";
+  version = "1.4.1";
+
+  src = fetchFromGitHub {
+    owner = "stackitcloud";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-fVe9PY5MFX6Yc31dLi6C1bXZ39Em+Piq4EWoQce8srY=";
+  };
+
+  vendorHash = "sha256-ifOKYXgIZcKT1jov5cGBtH3IdZBVBqHO/JNkMlDXops=";
+
+  postInstall = ''
+    mv $out/bin/kubectl-get-all $out/bin/kubectl-get_all
+  '';
+
```

Comment:
add versionCheckHook and nix-update-script

Reply:
Can you confirm that I have done that correctly now?